po/es.po

```diff
@@ -1597,40 +1590,40 @@ msgstr "atributo no encontrado"
 
 #: src/tagsouppullparser.cpp:145
 msgid "EOF found while reading XML tag"
-msgstr "EOF encontrado al leer la etiqueta XML"
+msgstr "Fin de archivo (EOF) encontrado al leer una etiqueta XML"
 
 #: src/urlviewformaction.cpp:48 src/urlviewformaction.cpp:61
 msgid "No link selected!"
-msgstr "¡Ningún enlace seleccionado!"
+msgstr "Ningún enlace seleccionado."
 
 #: src/urlviewformaction.cpp:140
 msgid "Save Bookmark"
 msgstr "Guardar marcador"
 
 #: src/urlviewformaction.cpp:162
 msgid "URLs"
-msgstr "URLs"
+msgstr "URL"
 
 #: src/view.cpp:493 src/view.cpp:519
 msgid "Error: feed contains no items!"
-msgstr "Error: ¡la fuente no contiene elementos!"
+msgstr "Error: la fuente no contiene elementos."
```

Comment:
Ditto about punctuation.

Reply:
Withdrawn, [see explanation in a comment below](https://github.com/newsboat/newsboat/pull/874#issuecomment-612408710).